### ISSUE TITLE
Add proper application startup

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#10]
### Who

- [ ] Anonymous visitor
- [ ] Patient
- [ ] Doctor
- [ ] Content editor
- [ ] Marketer
- [ ] Administrator
- [x] Developer

### Needs what

To properly register Xperience services required for the Essentials scenarios and use the services in the OWIN pipeline.

### Why

Why is the feature needed?

### Issue expands upon

This issue expands upon another issue(s): #`issue number`

### Blockers

This issue is blocked by the following issue(s): #`issue number`
